README.md

```diff
@@ -21,6 +21,12 @@ Create a database `participate_api_dev` and insert initial data:
 $ mix ecto.setup
 ```
 
+To migrate an existing database to a new version of the API server do this instead:
```

Comment:
Please amend my language use as you see fit. Unsure about the best wording here.